docs/fsharp/language-reference/compiler-options.md

```diff
@@ -15,53 +15,53 @@ The following table shows compiler options listed alphabetically. Some of the F#
-|**--warnon:&lt;int-list&gt;**|Enable specific warnings that might be off by default or disabled by another command line option. In F# 3.0, only the 1182 (unused variables) warning is off by default.|
-|**--warnaserror**[**+**&#124;**-**] [**&lt;int-list&gt;**]|Enables or disables the option to report warnings as errors. You can provide specific warning numbers to be disabled or enabled. Options later in the command line override options earlier in the command line. For example, to specify the warnings that you don't want reported as errors, specify **--warnaserror+ --warnaserror-:&lt;int-list&gt;**.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;warnaserror &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/406xhdz3.aspx).|
-|**--win32manifest:manifest-filename**|Adds a Win32 manifest file to the compilation. This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;win32manifest &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/bb545961.aspx).|
-|**--win32res:resource-filename**|Adds a Win32 resource file to the compilation.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;win32res (&#40;C&#35); Compiler Options&#41;](https://msdn.microsoft.com/library/8f2f5x2e.aspx).|
+|`-a filename.fs`|Generates a library from the specified file. This option is a short form of `--target:library filename.fs`.|
+|`--baseaddress:address`|Specifies the preferred base address at which to load a DLL.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;baseaddress &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/2fdbz5xd.aspx).|
+|`--codepage:id`|Specifies which codepage to use during compilation if the required page is not the current default codepage for the system.<br /><br />This compiler option is equivalent to the C# compiler option of the same name. For more information, see [&#47;codepage &#40;C&#35; Compiler Options&#41;](https://msdn.microsoft.com/library/w0kyekyh.aspx).|
+|`--consolecolors`|Specifies that errors and warnings use color-coded text on the console.|
+|<code>--crossoptimize[+&#124;-]</code>|Enables or disables cross-module optimizations.|
```

Comment:
I believe you could change this to use pipe and backticks now that we're using the markdig engine. same comment for the other rows where the pipe is being encoded.

```suggestion
|`--crossoptimize[+|-]`|Enables or disables cross-module optimizations.|
```